Saule/Http/NoDefaultIncludedAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Web.Http.Controllers;
+using System.Web.Http.Filters;
+
+namespace Saule.Http
+{
+    /// <summary>
+    /// Indicates that the action should not include related data by default.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method)]
+    public sealed class NoDefaultIncludedAttribute : ActionFilterAttribute
```

Comment:
Perhaps rename this `AllowsIncludesAttribute` and move that functionality from `AllowsQueryAttribute` into here? Let me know what you think. The documentation can still explain that this also disables the includes by default.